src/progress/progress_set.rs

```diff
@@ -64,6 +68,22 @@ impl Configuration {
             learners: learners.into_iter().collect(),
         }
     }
+
+    /// Create a new `ConfState` from the configuration itself.
+    pub fn to_conf_state(&self) -> ConfState {
```

Comment:
I take it there were some problems with the `From` implementations?

Reply:
Generally `From` needs to move the `Configuration`. Previously I write `impl From<&Configuration> for ConfState`, but Jay prefers `to_conf_state`.

Reply:
Yes I do too, that would create an implicit copy.